### DISCUSSION TITLE
MQTT Dynamic Power Limiter

### DISCUSSION THREAD

**Discussion by Feldsalat** [hoylabs/OpenDTU-OnBattery#922] [Q&A]
Kleines Problem... Ich habe zweit DTUs laufen und gestern jeweils ein Update gemacht. Es gibt jetzt in Home Assistant, bei MQTT, nun einen neuen Eintrag: "Dynamic Power Limiter". Es gibt aber nur einen Eintrag statt zwei und die Daten sind von beiden DTUs gemischt vorhanden. Basis-Topic ist jeweils unterschiedlich und kommt auch richtig an, es scheint mit Home Assistant MQTT-Auto-Discovery etwas falsch zu laufen. Mache ich etwas falsch?

**Comment by schlimmchen** [comment]
Schau doch mal in #973 und probiere die dortige [Firmware](https://github.com/helgeerbe/OpenDTU-OnBattery/actions/runs/9345876715?pr=973) die im Kontext dieses PR gebaut wurde. Ist das Problem damit behoben? Wäre super, wenn du Bildschirmfotos machen und teilen würdest.
